_posts/2023-12-14-cockpit-307.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Cockpit 307
+author: mvollmer
+date: '2023-12-15'
+tags: cockpit, machines
+slug: cockpit-307
+category: release
+summary: 'Storage redesign'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 307:
+
+
+## Storage redesign
+
+The Storage page has been redesigned with the goal of making more things visible in all views. The overview page now shows all storage objects in a table and allows almost all operations can be performed on them right from the table.
```

Comment:
"allows almost all operations can be" doesn't work.

either

"almost all operations can be..."

or

"allows all operations to be..."